homeassistant/components/homewizard/sensor.py

```diff
@@ -126,8 +126,9 @@
     ),
     SensorEntityDescription(
         key="gas_timestamp",
-        name="Gas Timestamp",
+        name="Last Gas Update",
```

Comment:
Why do we need this? Isn't this already visible because that's the time the gas usage state got updated in Home Assistant?

Reply:
I agree. Removed the sensor (f072a7d)